nilearn/plotting/data/js/surface-plot-utils.js

```diff
@@ -110,6 +110,7 @@ function getLayout(plotDivId, viewSelectId, blackBg) {
     let width = height * 3 / 2;
 
     let layout = {
+        showLegend: false,
```

Comment:
```suggestion
        showlegend: false,
```

Reply:
Sorry for not testing it, I've pushed the correction, legend is now off. Should I do a full circle build?

Reply:
thanks! no I think it's ok, I also checked it locally and that was the last missing detail